main.py

```diff
@@ -283,3 +292,71 @@ def _unpack_non_empty(desired_class, data):
         obj.unpack(data)
 
         return obj
+
+    def _get_interfaces(self):
+        """Get all interfaces."""
+        interfaces = []
+        switches = list(self.controller.switches.values())
+        for switch in switches:
+            interface_list = list(switch.interfaces.values())
+            interfaces = interfaces + interface_list
+        return interfaces
+
+    @staticmethod
+    def as_dict(interfaces):
+        """Return a dict of interfaces."""
+        interfaces_dic = {}
+        for interface in interfaces:
+            interfaces_dic[interface.id] = interface
+        return interfaces_dic
+
+    def _get_lldp_interfaces(self):
+        """Get interfaces enabled to receive the LLDP package."""
+        enabled_receive_lldp = []
+        interfaces = self._get_interfaces()
+        for interface in interfaces:
+            if interface.receive_lldp:
+                enabled_receive_lldp.append(interface.id)
+        return enabled_receive_lldp
```

Comment:
Maybe replace with a list comprehension?

Reply:
The code has been updated. List comprehension has been added.